config/initializers/searchkick.rb

```diff
@@ -0,0 +1,9 @@
+if defined?(Searchkick)
```

Comment:
Let's only do this in development or when using with Docker. In production communication to the queue should be secure.